src/tools/kdb/mount.c

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file
+ *
+ * @brief Implementation of kdb mount command
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include <command.h>
+#include <mount.h>
+
+#include <kdb.h>
+
+#include <cpp-main.h>
+
+#define COMMAND_NAME "mount"
+
+#define GET_OPTION_KEY(options, name) GET_OPT_KEY (options, COMMAND_BASE_KEY (COMMAND_NAME) "/" name)
+#define GET_OPTION(options, name) GET_OPT (options, COMMAND_BASE_KEY (COMMAND_NAME) "/" name)
+
+void addMountSpec (KeySet * spec)
+{
+	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME), KEY_META, "description", "Mount a new backend.", KEY_META, "command",
+				   COMMAND_NAME, KEY_END));
+
+	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/path", KEY_META, "description",
+				   "A filename (absolute for system, relative for cascading or user)", KEY_META, "args", "indexed",
+				   KEY_META, "args/index", "0", KEY_END));
+	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/mountpoint", KEY_META, "description",
+				   "where to mount the backend, start with / for cascading mp", KEY_META, "args", "indexed", KEY_META,
+				   "args/index", "1", KEY_END));
+
+	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/plugins/#", KEY_META, "description",
+				   "Plugin and its config, <PLUGIN>[key1=val1,key2=val2,...]", KEY_META, "args", "remaining", KEY_END));
```

Comment:
```suggestion
/* TODO: not supported by libelektra-opts right now
	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/path", KEY_META, "description",
				   "A filename (absolute for system, relative for cascading or user)", KEY_META, "args", "indexed",
				   KEY_META, "args/index", "0", KEY_END));
	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/mountpoint", KEY_META, "description",
				   "where to mount the backend, start with / for cascading mp", KEY_META, "args", "indexed", KEY_META,
				   "args/index", "1", KEY_END));

	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/plugins/#", KEY_META, "description",
				   "Plugin and its config, <PLUGIN>[key1=val1,key2=val2,...]", KEY_META, "args", "remaining", KEY_END));
*/
        // TODO: placeholder until the above is supported
	ksAppendKey (spec, keyNew (COMMAND_SPEC_KEY (COMMAND_NAME) "/args/#", KEY_META, "description",
				   "filename mountpoint plugins and config, see man page for details", KEY_META, "args", "remaining", KEY_END));
```

It won't generate such a nice help message, but it should accept all the stuff that `kdb mount` supports.